static/app/routes.tsx

```diff
@@ -2300,8 +2300,8 @@ function buildRoutes() {
 
   const organizationRoutes = (
     <Route component={errorHandler(OrganizationLayout)}>
-      <AutomationRoutes />
-      <DetectorRoutes />
```

Comment:
Turns out these couldn't be crawled properly 😔

Reply:
just curious, know what the limiting factor was? is it like we just needed to invoke the function here or is there something specific about exporting it as a const?

Reply:
Great question! Apparently, our route components aren't ever actually rendered... they are being crawled to construct a route tree that React Router can consume. 

This seems to have been done as part of the React Router 6 migration for backwards-compatability. https://github.com/getsentry/sentry/pull/78350